rules_gapic/java/java_gapic_pkg.bzl

```diff
@@ -171,8 +171,11 @@ def java_gapic_proto_gradle_pkg(
     _java_gapic_gradle_pkg(
         name = name,
         pkg_type = "proto",
+        # TODO: the output of `java_gapic_assembly_gradle_pkg` is broken now
+        #  (because of missing the maven artifacts for the Bazel dependencies
+        #  we now depend on (gax, grpc). Fix this.
         deps = deps + [
-            "@com_google_protobuf_protobuf_java//jar",
+            #            "@com_google_protobuf_protobuf_java//jar",
```

Comment:
This formatting is a bit odd... why not just remove these lines?

Reply:
Reformatted. I'm not removing those, because those deps should be put back eventually.